### DISCUSSION TITLE
Alt+F4 Close the Epoptes lock window

### DISCUSSION THREAD

**Discussion by GerardoSystem0o0** [epoptes/epoptes#65] [Q&A]
When you press "ALT + F4", while there is an active session for example students you can close the lock screen, however this does not happen if all users are disconnected.

Linux Mint 18.3 - MATE (64-bit) for server and clients

**Comment by alkisg** [comment]
Hi, Epoptes developers only speak Greek and English, not Spanish, could you rewrite your issue in English?

**Comment by lbssousa** [comment]
Basically, he says that pressing Alt+F4 on clients' computers closes lock screen, letting students bypass screen locking by teacher.

**Comment by GerardoSystem0o0** [comment]
Gracias lo corrigo de inmediato

**Comment by alkisg** [comment]
> Basically, he says that pressing Alt+F4 on clients' computers closes lock screen, letting students bypass screen locking by teacher.

Does it happen with the latest Epoptes version 1.0.0, available from the [Greek schools PPA](https://launchpad.net/~ts.sch.gr/+archive/ubuntu/ppa)?

**Comment by GerardoSystem0o0** [comment]
epoptes-client v 0.5.10-1 is the installed version and according to the system it is the last one.

**Comment by alkisg** [comment]
The older 0.5.10 Epoptes version is based on Python2/Gtk2 and will not receive any updates.

I uploaded the newer 1.0.0 version in the [Epoptes proposed PPA](http://www.epoptes.org/documentation/ppa) for Ubuntu 18.10, 18.04 and 16.04.
If you still have a the problem with that version, I could take a look.

**Comment by GerardoSystem0o0** [comment]
I can not install epoptes 1.0.0 in linux mint.
Did they release the version for ubuntu?

**Comment by alkisg** [comment]
Yes, I uploaded it to Ubuntu 18.10.
As I said, you can use the PPA for Linux Mint or any other Ubuntu-based distro.

**Comment by GerardoSystem0o0** [comment]
add to linux mint
add-apt-repository ppa: epoptes
however, it does not detect the most recent version.

**Comment by GerardoSystem0o0** [comment]
if you wish I can give you access with teamviewer, so you can verify that epoptes is not updated.
I do not think I'm doing something wrong

**Comment by alkisg** [comment]
Which Ubuntu version is your Linux Mint version?

**Comment by GerardoSystem0o0** [comment]
DISTRIB_ID=LinuxMint
DISTRIB_RELEASE=18.3
DISTRIB_CODENAME=sylvia
DISTRIB_DESCRIPTION="Linux Mint 18.3 Sylvia"
NAME="Linux Mint"
VERSION="18.3 (Sylvia)"
ID=linuxmint
ID_LIKE=ubuntu
PRETTY_NAME="Linux Mint 18.3"
VERSION_ID="18.3"
HOME_URL="http://www.linuxmint.com/"
SUPPORT_URL="http://forums.linuxmint.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/linuxmint/"
VERSION_CODENAME=sylvia
UBUNTU_CODENAME=xenial


Linux version 4.15.0-36-generic (buildd@lcy01-amd64-017) (gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.10)) #39~16.04.1-Ubuntu SMP Tue Sep 25 08:59:23 UTC 2018

**Comment by GerardoSystem0o0** [comment]
already solve it, I downloaded manually from the following url:

EPOPTES-CLIENT
http://launchpadlibrarian.net/391517917/epoptes-client_1.0.1-1_all.deb

EPOPTES-MASTER
http://launchpadlibrarian.net/391517919/epoptes_1.0.1-1_all.deb

I hope it helps others

**Comment by GerardoSystem0o0** [comment]
the problem is solved with the update, you can close the theme
